cmd/kpod/spec.go

```diff
@@ -110,8 +126,20 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		configSpec.Linux.Seccomp = &seccompConfig
 	}
 
+	configSpec.Process.Env = config.env
```

Comment:
Does the spec generation have a way to add environment and bind/tmpfs mounts? If so maybe we should add a TODO to convert to using that at some point. I don't think we need to migrate to it right away.

Reply:
Yes, I will convert those over as well.